AuxTel/AuxTel-Troubleshooting/General-Troubleshooting/AT-Elevation-out-of-range.rst

```diff
@@ -0,0 +1,160 @@
+.. This is a template for Image out of focus when some part of the observatory enters an abnormal state. This comment may be deleted when the template is copied to the destination.
+
+.. Review the README in this procedure's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *I. Sotuela*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _AuxTel-AT-Elevation-out-of-range:
+.. Each section should include a label for cross-referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with a reST object such as a title or figure, you must include the link with an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+######################################
+AT Elevation Out of range
+######################################
+
+.. note::
+    This is a procedure template file that is associated with a template directory structure. This note should be deleted when the section is properly populated.
+
+.. _AT-Elevation-out-of-range-Overview:
+
+Overview
+========
+
+.. In one or two sentences, explain when this AT correct_pointing failure procedure needs to be used. Describe the symptoms that the user sees to use this procedure.
+
+This procedure should be used when the AT (Auxiliary Telescope) `correct_pointing` script fails with the error `Rejected: elevation out-of-range`.
```

Comment:
Include the full path to the SAL Script and use the format https://obs-ops.lsst.io/project/observatory-operations-reST-reference.html#filenames-and-paths

Reply:
`auxtel/correct_pointing.py`